diff --git a/wezterm-toast-notification/build.rs b/wezterm-toast-notification/build.rs
index 5ecc6da67..cea919d6d 100644
--- a/wezterm-toast-notification/build.rs
+++ b/wezterm-toast-notification/build.rs
@@ -1,5 +1,5 @@
 fn main() {
-    if cfg!(target_os = "macos") {
-        println!("cargo:rustc-link-lib=framework=UserNotifications");
-    }
+    // if cfg!(target_os = "macos") {
+    //     println!("cargo:rustc-link-lib=framework=UserNotifications");
+    // }
 }
